app/javascript/components/ingredients/ingredients-container.vue

```diff
@@ -385,6 +411,57 @@ export default {
       const objectIndex = this.ingredients.findIndex((obj => obj.id === this.ingredientToEdit.id));
       Vue.set(this.ingredients, objectIndex, ingredientEdited);
     },
+
+    // eslint-disable-next-line max-statements,complexity
+    validations(form) {
+      this.errors = {
+        name: '',
+        quantity: '',
+        measure: '',
+        price: '',
+        minimumQuantity: '',
+      };
+      let validForm = true;
+
+      const quantity = form.ingredientMeasuresAttributes[0].quantity;
+
+      if (!(typeof (quantity - 0) === 'number') || !(quantity > 0)) {
+        this.errors.quantity = 'floatNonZero';
+        validForm = false;
```

Comment:
en vez de usar esto al final de cada uno de los `if`, se podría hacer esto al final del método:

```js
const validForm = Object.values(this.errors).some(value => !!value);
return validForm;
```

El double bang se explica [acá](https://betterprogramming.pub/javascript-bang-bang-i-shot-you-down-use-of-double-bangs-in-javascript-7c9d94446054#:~:text=Gotchas!%20Oh%2C%20No!%20I%20Shot%20You%20Down!). Te tinca?

Reply:
bacan! me resultó. Eso si tenía que ser `!(Object.values(this.errors).some(value => !!value))` para que funcionara bien, pero buenisima, gracias!